uber/site_sections/registration.py

```diff
@@ -32,7 +32,7 @@ def checking_at_the_door(self, *args, **kwargs):
 
 @all_renderable(c.PEOPLE, c.REG_AT_CON)
 class Root:
-    def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first', invalid=''):
+    def index(self, session, message='', page='1', search_text='', uploaded_id='', order='last_first', invalid=''):
```

Comment:
We could add a button when the page is displayed blank that says click to load first page of all results or something.. Not sure if that's needed, if this is just for testing then hitting search on an empty search box / clicking "1" is all you need to do.

> On Jan 19, 2016, at 12:26 PM, Dominic Cerquetti <notifications@github.com> wrote:
> 
> In uber/site_sections/registration.py:
> 
> > @@ -32,7 +32,7 @@ def checking_at_the_door(self, *args, **kwargs):
> >  
> >  @all_renderable(c.PEOPLE, c.REG_AT_CON)
> >  class Root:
> > -    def index(self, session, message='', page='0', search_text='', uploaded_id='', order='last_first', invalid=''):
> > +    def index(self, session, message='', page='1', search_text='', uploaded_id='', order='last_first', invalid=''):
> ok if we're worried about perf (reasonable) then yea forget it.
> 
> we do I think already do what you're saying below:
> 
>          if search_text:
>              page = page or 1
> —
> Reply to this email directly or view it on GitHub.
>